packages/custom-storage/__tests__/SQLStorage.spec.ts

```diff
@@ -0,0 +1,179 @@
+import { SQL } from '@carto/toolkit-sql';
+import { SQLStorage } from '../src/sql/SQLStorage';
+import {
+  StoredVisualization,
+  // Visualization,
+  // Dataset
+} from '../src/StorageRepository';
+import {
+  CHECK_CREATE_QUERIES,
```

Comment:
Nice!
I sometimes do that kind of extraction, specially if the queries are going to be reused. If they are not, they could also be inside the test itself (they are not so long after all...), to avoid an indirection, but that's just my preference

Reply:
Totally right, I'll change it!